packages/near-membrane-base/src/red.ts

```diff
@@ -14,7 +14,6 @@ import {
     BlueArray,
```

Comment:
in this file we pretty much did the same we did in blue.ts, they are now more alike.